dev-packages/cli/src/generator/backend-generator.ts

```diff
@@ -38,16 +37,18 @@ function load(raw) {
     )
 }
 
-function start(port, host) {
+function start() {
+    const cliManager = container.get(CliManager);
+    cliManager.initializeCli();
     const application = container.get(BackendApplication);
     application.use(express.static(path.join(__dirname, '../../lib'), {
         index: 'index.html'
     }));
-    return application.start(port, host);
+    return application.start();
 }
 
-module.exports = (port, host) => Promise.resolve()${this.compileBackendModuleImports(backendModules)}
-    .then(() => start(port, host)).catch(reason => {
+Promise.resolve()${this.compileBackendModuleImports(backendModules)}
```

Comment:
`module.exports = `? otherwise the port won't be propagated to the electron renderer process

Reply:
also wdio tests pass args programmatically